modules/geo-layers/src/h3-layers/h3-hexagon-layer.js

```diff
@@ -103,6 +102,14 @@ export default class H3HexagonLayer extends CompositeLayer {
     this.setState({centerHex: hex, vertices});
   }
 
+  getSubLayerAccessor(accessor) {
```

Comment:
Resolved it.

Reply:
Resolved the issue.

Reply:
It is also present in the H3Cluster Layer. Should I change it over there as well?

Reply:
If you look closely the H3ClusterLayer version returns something different:

```js
(object, objectInfo) => accessor(object.object, objectInfo)
```

This is because the H3ClusterLayer does not pass the `data` prop from the user directly to the sublayer. Instead, it creates an internal `data` array. In order for the user's accessors to see the original data object, we need to "unwrap" the generated data.

Reply:
Makes sense. Thank you.